diff -u1 -rN a/Source/WebKit/qt/declarative/experimental/experimental.pri b/Source/WebKit/qt/declarative/experimental/experimental.pri
--- a/Source/WebKit/qt/declarative/experimental/experimental.pri	2016-03-18 00:51:28.547089614 +0000
+++ b/Source/WebKit/qt/declarative/experimental/experimental.pri	2016-03-18 10:47:29.467636377 +0000
@@ -52,2 +52,2 @@
 
-
+INCLUDEPATH += $${ROOT_WEBKIT_DIR}/include/QtWebKit/
diff -u1 -rN a/Source/WebKit/qt/declarative/public.pri b/Source/WebKit/qt/declarative/public.pri
--- a/Source/WebKit/qt/declarative/public.pri	2016-03-18 00:51:28.547089614 +0000
+++ b/Source/WebKit/qt/declarative/public.pri	2016-03-18 10:46:05.699286973 +0000
@@ -55 +55,3 @@
 INSTALLS += qmldir
+
+INCLUDEPATH += $${ROOT_WEBKIT_DIR}/include/QtWebKit/
